src/gulpfile.js

```diff
@@ -73,8 +73,18 @@ gulp.task('generate-resjson', (cb) => {
     runSequence(['generate-resjson-json', 'generate-resjson-interface'], 'merge-localized-json', cb);
 });
 
+gulp.task('generate-angular-cli-json', (_) => {

```

Comment:
Please move this new task definition above the the task definition of generate-powershell to match the order of runSequence(['generate-angular-cli-json', 'generate-powershell', 'generate-svg', 'generate-resjson'], cb).

Reply:
done